webapp/channels/src/sass/routes/_admin-console.scss

```
@@ -619,49 +625,68 @@
     min-height: 0;
     flex-direction: column;
-    background: #111;
+    background: var(--sys-sidebar-bg);
+    color: var(--sys-sidebar-text);
     grid-area: lhs;
 
     .filter-container {
         position: relative;
+        z-index: 1;
+        margin-top: 8px;
+        padding: 0px 12px;
```

Stylelint (17.14.0)

[error] Expected "padding" to come before "margin-top" (order/properties-order)

(order/properties-order)